overlord/servicestate/quota_control.go

```diff
@@ -86,13 +88,29 @@ func validateSnapForAddingToGroup(st *state.State, name string, allGrps map[stri
 	return nil
 }
 
+func quotaGroupsAvailable(st *state.State) error {
+	tr := config.NewTransaction(st)
+	enableQuotaGroups, err := features.Flag(tr, features.QuotaGroups)
+	if err != nil && !config.IsNoOption(err) {
+		return err
+	}
+	if !enableQuotaGroups {
+		return fmt.Errorf("cannot create quota group: please enable the experimental.quota-groups option")
```

Comment:
Maybe this should be more generic to allow using in different context, e.g. "cannot manipulate quota group: ..."?

Reply:
For parallel installs we use the following message:
```
experimental feature disabled - test it by setting 'experimental.parallel-instances' to true
```

Reply:
Good idea, this seems to be the pattern used in multiple places.

Reply:
Pushed.